coralproject/talk
# Focusing on reply box when reply is clicked.

## What does this PR do?

Automatically switches focus to the reply box when the reply button is clicked.

## How do I test this PR?

1. Click the reply button.
2. Start typing.
3. Text should appear in the reply box.
